packages/dashboard-message-bus/lib/utils.ts

```diff
@@ -0,0 +1,141 @@
+import WebSocket, { ServerOptions } from "isomorphic-ws";
+import { Message, PortsConfig } from "./types";
+import any from "promise.any";
+import delay from "delay";
+import axios from "axios";
+
+any.shim();
```

Comment:
Is `Promise.any` unnavailable?

Reply:
`Promise.any` is a Node 15 feature so we can't rely on it directly since we have to support Node 12.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any#browser_compatibility